ui/src/app/services/cohort-edit.service.ts

```diff
@@ -0,0 +1,49 @@
+// Data service used to pass information from the edit cohort page
+// to the workspace page.
+
+// TODO (blrubenstein): Remove this service once edits to cohorts
+// are done server side.
+
+import {Injectable} from '@angular/core';
+
+import {Cohort} from 'generated';
+
+@Injectable()
+export class CohortEditService {
```

Comment:
Might try to update these signature on thus service's methods to match the real-time CohortService, and then supply this one to the injector as an alternative.